app/lib/features/pins/providers/notifiers/create_pin_notifier.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:acter/features/pins/models/create_pin_state/create_pin_state.dart';
+import 'package:acter/features/pins/models/create_pin_state/pin_attachment_model.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class CreatePinNotifier extends StateNotifier<CreatePinState> {
+  final Ref ref;
+
+  CreatePinNotifier({required this.ref}) : super(const CreatePinState());
```

Comment:
why are you requesting ref? You don't need ref.

Reply:
I was experimenting something and forgot to remove it. Good catch. Let me remove it.